src/pre_defined_classes/mathematical_program/medium/q1/MainTest.java

```diff
@@ -0,0 +1,32 @@
+package pre_defined_classes.mathematical_program.medium.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The BMI of a person that weighs "),
+                new DoubleLiteral("weight"),
+                new StringLiteral(" kilograms and is "),
+                new DoubleLiteral("height"),
+                new StringLiteral(" metres tall is: "),
+                new DoubleLiteral(26, 27, "BMI")
```

Comment:
If it's outside this range it's not structurally wrong, being outside of this range is something to check in the next test